File: website/docs/code-samples/rt-preview-api-setup.m
CrowdinProviderConfig *crowdinProviderConfig = [[CrowdinProviderConfig alloc]
    initWithHashString:@"{your_distribution_hash}"
    sourceLanguage:@"{source_language}"
    organizationName:@"{organization_name}"];

CrowdinSDKConfig *config = [[[[CrowdinSDKConfig config]
    withCrowdinProviderConfig:crowdinProviderConfig]
    withRealtimeUpdatesEnabled:YES]
    // highlight-next-line
    withAccessToken:@"your_access_token"];

File: website/docs/code-samples/rt-preview-api-setup.swift
let crowdinProviderConfig = CrowdinProviderConfig(hashString: "{your_distribution_hash}",
    sourceLanguage: "{source_language}",
    organizationName: "{organization_name}") // Optional. Required for Enterprise only

let crowdinSDKConfig = CrowdinSDKConfig.config()
    .with(crowdinProviderConfig: crowdinProviderConfig)
    .with(realtimeUpdatesEnabled: true)
    // highlight-next-line
    .with(accessToken: "your_access_token")
    .with(settingsEnabled: true)
